blocks/donor-wall/class-give-donor-wall.php

```diff
@@ -173,8 +169,24 @@ public function register_block() {
 					],
 					'avatarSize'      => [
 						'type'    => 'string',
-						'default' => '60',
+						'default' => '75',
 					],
+                    'toggleOptions'      => [
+                        'type'    => 'string',
+                        'default' => 'Donor info',
+                    ],
+                    'filter'      => [
+                        'type'    => 'string',
+                        'default' => 'Donor ID',
```

Comment:
@JoshuaHungDinh I think it is not the correct default value for the filter, it should be `ids`.